src/formats/GRO.cpp

```diff
@@ -64,9 +64,9 @@ void GROFormat::read_next(Frame& frame) {
             );
         }
 
-        size_t resid = SIZE_MAX;
+        int64_t resid = std::numeric_limits<int64_t>::max();
```

Comment:
Side note, but this looks like a good use case for `optional<int64_t>`. We should refactor at some point =)

Reply:
It probably should be addressed elsewhere, it's a really easy change!